src/api/hub.ts

```diff
@@ -0,0 +1,31 @@
+import axios from 'axios';
+
+export default class HubGroupAPI {
+  private readonly defaultBaseUrl = 'https://63d6be0e-bd49-4c76-8fe5-195bb7cf88a5.mock.pstmn.io';
+  private readonly baseUrl: string;
+
+  private readonly apiPath = '/ent/api/templates/bundlegroups';
+
+  constructor(baseUrl?: string) {
+    this.baseUrl = baseUrl || this.defaultBaseUrl;
+  }
+
+  async callApi(uri: string): Promise<any[]> {
+    const response = await axios(`${this.baseUrl}${uri}`);
+    const { data } = response;
+
+    return data;
+  }
+
+  getHubGroups(): Promise<any[]> {
+    return this.callApi(this.apiPath);
+  }
+
+  getHubGroupById(versionId: number): Promise<any[]> {
+    return this.callApi(`${this.apiPath}/${versionId}`);
+  }
+
+  getHubGroupByName(name: string): Promise<any[]> {
```

Comment:
this should be merged with getBundleGroups, we don't need two methods, just an optional name parameter to pass to api. Also use this form:
```
// Equivalent to `axios.get('https://httpbin.org/get?answer=42')`
const res = await axios.get('https://httpbin.org/get', { params: { answer: 42 } });
```

Reply:
done; merged to `getBundleGroups`